### DISCUSSION TITLE
Moderation profiles

### DISCUSSION THREAD

**Discussion by alexbo95** [Chatterino/chatterino2#5418] [Ideas]
It would be great to have moderation profiles, especially for Moderation buttons. Across channels, different buttons & actions are required and it would be great to safe these profiles and apply them in different channels. Definitely helpful for mods across many channels, especially large ones where commands are necessary. 

Different profile options exampled in pictures attached (would be nice to select a certain profile and apply it to a channel or be able to switch between profiles)

![chatprof2](https://github.com/Chatterino/chatterino2/assets/17392292/14bbb4a1-abcd-49cd-a2cd-4eb71eb77d71)
![chatprof1](https://github.com/Chatterino/chatterino2/assets/17392292/f6ee8ffb-5e4c-4bf8-9a74-cae1818de13e)

**Comment by pajlada** [comment]
Would a reasonable solution be to select, per moderation button, if it should be enabled in all channels or only a subset of channels?
